_shared/hooks/post_gen_project.py

```diff
@@ -19,6 +19,10 @@ def remove_conditional_files():
     paths_to_remove.extend(["package.json", "yarn.lock"])
     {% endif %}
 
+    {% if cookiecutter.get("devdata") != "yes" %}
+    paths_to_remove.extend(["bin/make_devdata"])
+    {% endif %}
```

Comment:
This deletes the `bin/make_devdata` file from the cookiecutter's output for projects that don't have devdata enabled, using the same technique that the cookiecutter uses for all such optional files.

Note that when updating an existing project with `make template` this deletes `bin/make_devdata` _from the temporary directory_ that the cookiecutter's output has been written to before walking that temporary directory and moving files into the project. So if the project has devdata disabled and has its own `bin/make_devdata` file then `make template` _will not_ overwrite that file.